nntrainer/layers/activation_layer.cpp

```diff
@@ -49,26 +49,25 @@ int ActivationLayer::initialize() {
 sharedConstTensors ActivationLayer::forwarding(sharedConstTensors in) {
   input = *in[0];
   /// @note @a _act_fn is expected to work out of place and not modify @a input
-  hidden = _act_fn(input);
+  hidden = _act_fn(input, hidden);
```

Comment:
Just confirming I guess no need for the assignment?

Reply:
done.